content-services/content-sources-backend
# Fixes 4445: add apis to bulk import/export custom repos

## Summary

Adds bulk import and export apis for custom repos

## Testing steps

1. Add a couple custom repos and let them snapshot
2. Make a request to export the repos and send the repository uuids in the body: `POST /repositories/bulk_export/`
3. Use the response you get from the bulk export (try changing some of the fields) to make a request to import the repos into the same org (`POST /repositories/bulk_import/`). You should see warnings in the response that the repo urls already exist and any mismatches between fields of existing and new repos
4. Use the same response to import the repos into a different org that doesn't have those repos. The repos should be created and introspection and snapshot tasks should be triggered
